app/helpers/accounts_helper.rb

```diff
@@ -0,0 +1,11 @@
+module AccountsHelper
+  def formatted_currency(value)
+    ActiveSupport::NumberHelper.number_to_currency(
+      value/100.0,
+      unit: 'R$',
+      separator: ',',
+      delimiter: '.',
+      format: "%u %n"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.

[Read more about it here.](http://www.rubydoc.info/github/bbatsov/rubocop/Rubocop/Cop/Style/StringLiterals)